src/components/StatusBar/StatusBar.jsx

```diff
@@ -1,37 +1,56 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import './status-bar.scss'
 
-const StatusBar = ({ statusArray, apiRequestStatus, addClass }) => (
-  <div className={`container ${addClass}`}>
```

Comment:
the majority of this code is the same. I needed to create a new variable and function though, so the return body is no longer immediately being returned. a lot of this change is adding indention for that reason.